.github/workflows/maintenance.md

```diff
@@ -0,0 +1,24 @@
+# Maintenance
+
+## How to update supported Kubernetes
+
+- Update Kubernetes version in `Makefile.versions`.
+- Update `k8s.io/*` packages version in `go.mod`.
+
+## How to update supported Flatcar Container Linux
+
+NecoPerf supports one Flatcar Linux version at a time.
+If a new stable version of Flatcar Linux is released, please do the following steps.
+
+- Check the [stable release](https://www.flatcar.org/releases) of Flatcar Linux
+- Find the tags for the container image of the [flatcar-sdk-amd64](https://github.com/orgs/flatcar/packages/container/package/flatcar-sdk-amd64) corresponding to the stable version of Flatcar Linux
+- Update `FLATCAR_VERSION` in `Makefile.versions`.
+
+If Flatcar Container Linux Container Image has changed, please fix the relevant source code.
```

Comment:
What does it mean?
It seems that this "If" and previous "If" are same.

Reply:
How about adding examples?

```
1. Check ...
    * e.g. "3602.2.2" is released as Stable.
2. Find ...
    * e.g. "3602.0.0" is the corresponding tag.
    * Note that the container images of the flatcar-sdk-amd64 are not updated as frequent as  the Flatcar releases.
3. Update ...
    * e.g. Use "3602.0.0" as `FLATCAR_VERSION`.
```

Reply:
```suggestion
If the contents of the container image have changed, especially in terms of the files layout, please update the relevant Dockerfiles.
```

Reply:
s/frequent/frequently/